roles/provision-unifiedpush-apb/defaults/main.yml

```diff
@@ -5,8 +5,8 @@ ups_port: 8080
 ups_proxy_port: 4180
 
 # Non global constants
-postgres_image: centos/postgresql-95-centos7
-postgres_image_tag: '9.5'
+postgres_image: docker.io/centos/postgresql-96-centos7
```

Comment:
Would it use `docker.io` by default if not provided (as in the old code)?

Reply:
yes, sort of... if you not use docker CLI from upstream docker, it may be the case that your distribution goes to a different ORG before (e.g. `fedoraproject`)

Reply:
also, it seems to be best practice to use the fqn of the image:

* https://github.com/ansibleplaybookbundle/eclipse-che-apb/blob/master/apb.yml#L14-L16
* https://github.com/ansibleplaybookbundle/mssql-apb/blob/master/apb.yml#L13
* https://github.com/ansibleplaybookbundle/mediawiki-apb/blob/master/apb.yml#L9
to name only a few